python/pyspark/rdd.pyi

```diff
@@ -60,6 +60,7 @@ from pyspark.sql._typing import RowLike
 from py4j.java_gateway import JavaObject  # type: ignore[import]
 
 T = TypeVar("T")
+T_ = TypeVar("T_", covariant=True)
```

Comment:
Not sure the name `T_` is commonly used though - haven't checked other references. Otherwise LGTM

Reply:
NumPy and some of the typeshed annotations use `_T`, `_T_co` and `_T_contra` for invariant, covariant, and contravariant respectively. It is a bit verbose for my taste, but otherwise looks good.

Other projects I've seen (dry-python, django-stubs, pfun) don't use any special convention. Not the way I'd go :)

Reply:
I'll add `_co` suffix on provisional basis ‒ if we decide on different strategy, we can adjust thing in a separate PR.